packages/asset-swapper/package.json

```diff
@@ -52,6 +52,7 @@
         "@0x/json-schemas": "^5.0.7",
         "@0x/order-utils": "^10.2.4",
         "@0x/orderbook": "^2.2.5",
+        "@0x/quote-server": "^1.0.0",
```

Comment:
Is the plan to merge & publish https://github.com/0xProject/quote-server/pull/5 and then bump up the version in this PR?

Reply:
It looks like once this happens, we can modify many `IndicativeQuote` types to be more explicitly `RFQTIndicativeQuote` types, correct?

Reply:
Yes.  The that quote-server version has been published, and this PR now refers to it (via [this commit](https://github.com/0xProject/0x-monorepo/pull/2582/commits/b91cc3781de259e8647792ccd51db688419f7309)), and the more  RFQT-specific types are now in use, via [this commit](https://github.com/0xProject/0x-monorepo/pull/2582/commits/ad868af96e13f3efa68d5836d898c53a6a5f8252) and [this one](https://github.com/0xProject/0x-monorepo/pull/2582/commits/7162935028a676a93d98e1360be07a7122fb5e56).